docs/modules/operation/pages/operation/newts/introduction.adoc

```diff
@@ -2,4 +2,11 @@
 [[ga-opennms-operation-newts]]
 = Newts
 
-This section describes how to configure _{page-component-title}_ to use _Newts_ and how to use _{page-component-title}_ to monitor your Cassandra cluster.
+link:http://newts.io/[Newts] is a time-series data schema for link:http://cassandra.apache.org/[Apache Cassandra].
+It enables link:https://en.wikipedia.org/wiki/Scalability#Horizontal[horizontally scale] capabilities for your time series storage and is an alternative to link:http://www.opennms.org/wiki/JRobin[JRobin] and link:http://oss.oetiker.ch/rrdtool/[RRDtool].
```

Comment:
Possibly [horizontal scaling] for link text. Also, may want to have JRobin link to a non-wiki site, to remove reliance on that site.